compiler/x/codegen/X86SystemLinkage.cpp

```diff
@@ -283,9 +283,9 @@ TR::X86SystemLinkage::copyParametersToHomeLocation(TR::Instruction *cursor)
 TR::Instruction *
 TR::X86SystemLinkage::savePreservedRegisters(TR::Instruction *cursor)
    {
-   // For IA32, if disableShrinkWrapping, usePushForPreservedRegs will be true; otherwise false;
-   //          For X64,  shrinkWraping is always on, and usePushForPreservedRegs always false;
-    // TR_ASSERT(!getProperties().getUsesPushesForPreservedRegs(), "assertion failure");
+   // For IA32 usePushForPreservedRegs will be true; otherwise false;
+   // For X64, usePushForPreservedRegs always false;
+   // TR_ASSERT(!getProperties().getUsesPushesForPreservedRegs(), "assertion failure");
```

Comment:
Please remove this commented ASSERT too.

Reply:
Fixed in ed49592